pkg/trace/agent/agent.go

```diff
@@ -394,21 +390,19 @@ func (a *Agent) runSamplers(pt ProcessedTrace, hasPriority bool) (bool, float64)
 // samplePriorityTrace samples traces with priority set on them. PrioritySampler and
 // ErrorSampler are run in parallel. The ExceptionSampler catches traces with rare top-level
 // or measured spans that are not caught by PrioritySampler and ErrorSampler.
-func (a *Agent) samplePriorityTrace(pt ProcessedTrace) (sampled bool, rate float64) {
-	sampledPriority, ratePriority := a.PrioritySampler.Add(pt)
-	if traceContainsError(pt.Trace) {
-		sampledError, rateError := a.ErrorsScoreSampler.Add(pt)
```

Comment:
Previously we were calling `Add` on both PrioritySampler and ErrorSampler. Adding a note in case it changes somehting

Reply:
This will increase slightly error sampling rates as priority > 0 won't be taken in account.